### PR TITLE
Patch source to enable building with any LAPACK provider

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~~Reset the build number to `0` (if the version changed)~~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

See https://github.com/grimme-lab/crest/pull/78

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,28 +8,33 @@ package:
...
     - llvm-openmp  # [osx]
   host:
-    - mkl
-    - mkl-include
+    - libblas
+    - liblapack
   run:
-    - mkl
+    - libblas
+    - liblapack
```
Comment (jan-janssen) [inline recipe/meta.yaml:37]: It should be fine to only mention `libblas` and `liblapack` in the `host` section https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=openmp#blas

**Comment by jan-janssen** [comment]
In addition if you are interested, feel free to add yourself to the maintainers.

File: recipe/meta.yaml (hunk 1 above)
Comment (awvwgk) [reply recipe/meta.yaml:37]:
```suggestion
```

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Pull request merged**